bin/ghe-restore

```diff
@@ -341,6 +341,16 @@ else
   fi
 fi
 
+# Clean up all stale replicas
+if ! $CLUSTER; then
+  inactive_nodes=$(echo "ghe-spokes server show inactive --json | jq -r '.[] | select(.host | contains(\"git-server\")).host' | sed 's/^git-server-//g'" | ghe-ssh "$GHE_HOSTNAME" -- /bin/bash)
```

Comment:
Unfortunately this fails to cleanup the stale HA nodes if the target instance is in an unconfigured state (and the replica(s) were online at the time of the backup).

This occurs because when unconfigured, we don't automatically run `ghe-config-apply` near the end of the restore, which results in the nodes still appearing as `online` in the database. It takes a `ghe-config-apply` for these stale nodes to transition to a `offline` state.

As an alternative, could we drop the `inactive` from the `ghe-spokes server show` and instead just cleanup any nodes that don't match the UUID that we restored (as found in `$GHE_RESTORE_SNAPSHOT_PATH/uuid`)?

Reply:
Ooh, good point. I completely forgot about restoring to an unconfigured instance. Your suggestion makes sense too.